### ISSUE TITLE
wrong value for update_in field

### ISSUE THREAD

**Issue by ktruehl** [techdivision/import-category-ee#34]
Hi again!

When creating categories the import uses a rather arbitrary value for the `updated_in` field in the `catalog_category_entity` table:
```
MemberNames::UPDATED_IN => strtotime('+20 years')
```

Instead the import should use the same value Magento is using:
```
MemberNames::UPDATED_IN => \TechDivision\Import\Ee\Utils\SqlConstants::MAX_UNIXTIMESTAMP
```

The same goes for the EE product import, but it doesn't make much sense to create an additional ticket, unless you want me to.

Here are the patches we're using:
[bugfix-[17.0]--use-correct-value-for-updated-in.diff.zip](https://github.com/techdivision/import-category-ee/files/3534769/bugfix-.17.0.--use-correct-value-for-updated-in.diff.zip)
[bugfix-[16.0]--use-correct-value-for-updated-in.diff.zip](https://github.com/techdivision/import-category-ee/files/3534770/bugfix-.16.0.--use-correct-value-for-updated-in.diff.zip)

Best,
Klaas

**Comment by wagnert** [comment]
Hi @ktruehl Thank's for reporting. We'll have a look at it as soon as possible.

**Comment by wagnert** [comment]
@ktruehl Has been fixed with version 3.7.3. Thanks for the patches, they are well appreciated.
